### PR TITLE
SSL cert checking disabled

### PR DESCRIPTION
**Pull request by iskanderbs**
Fix for error: x509: certificate signed by unknown authority

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/xanzy/go-gitlab
+module github.com/iskanderbs/go-gitlab
```
Comment (svanharmelen) [inline go.mod]: Uuh... Please don't change this 😏

Comment (iskanderbs) [reply go.mod]: Sorry for my stupid solution, thank you for your reply)

Comment (svanharmelen) [reply go.mod]: No worries, all good 👍

**Comment by colynn** [comment]
Defined new http.Client as below:
```
gitlabToken := token
transCfg := &http.Transport{
  TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
}
httpClient := &http.Client{
      Transport: transCfg,
}

client := go-gitlab.NewClient(httpClient, gitlabToken)
```

**Pull request closed without merging**